src/utils/reducers/stream.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Set the stream context.
+ */
+export const streamSet = function(state, action) {
```

Comment:
Creating new reducer files as to reduce the size of `redux.js` (no pun intended!)